todo_app/templates/index.html

```diff
@@ -2,17 +2,87 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
+<div class="jumbotron">
+  <h1 class="display-4">To-Do App</h1>
+  <p class="lead">Just another to-do app.</p>
+</div>
+
+{% include "new_task_form.html" %}
```

Comment:
This is the main issue - I think you've forgotten to commit the file you extracted this into! I was able to get your app running by using the version that you removed from this template and creating the file myself, but we should definitely do that 🙂

Reply:
Sorry about that! `git add -p` letting me down!